actions/icsp_id_to_os.yaml

```diff
@@ -0,0 +1,38 @@
+---
+name: icsp_id_to_os
+runner_type: orquesta
+pack: hpe_icsp
+enabled: true
+entry_point: workflows/icsp_id_to_os.yaml
+description: Apply OS Build plans to a single Machine Serial Number/UUIDs. This will create a batch job in ICSP for the specified server.
+
+parameters:
+  identifier:
+    required: true
+    position: 0
+    type: string
+    description: Serial Number or UUID of the target Server.
+  id_type:
+    required: true
+    position: 1
+    type: string
+    enum:
+      - serialnumber
+      - UUID
+    default: serialnumber
+  hostname:
+    required: true
+    position: 2
+    type: array
```

Comment:
Isn't this now a single string being passed not an array. And therefore also the description doesn't need the comment about "Must match order of identifiers"